app/screens/thread_options/options/copy_link_option.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
There are
2 options are different for `thread options`
4 options common for both `thread options` and `post options`
5 options are different for `post options`

Instead of having all of them at one place, I'm creating a component `common_post_options` and moving the commonly used options there: `base_option`, `reply_option`, `copy_permalink_option`, `save_option`.